.pipelines/pipeline.yaml

```diff
@@ -426,7 +426,7 @@ stages:
       variables:
         commitID: $[ stagedependencies.setup.env.outputs['EnvironmentalVariables.commitID'] ]
       jobs:
-        - job: delete
+        - job: delete_build
```

Comment:
Does updating this name have any other side-effects? Is this stage references anywhere else?

Reply:
This is the last job in the last stage. It does not get referenced anywhere. The main purpose for adding the `_build` and `_test` is so that AZP knows that these are different jobs.